DataFormats/Detectors/CTP/src/Configuration.cxx

```diff
@@ -780,6 +780,15 @@ int CTPConfiguration::getInputIndex(const std::string& name) const
   LOG(info) << "input:" << name << " index:" << index;
   return index;
 }
+std::string CTPConfiguration::getClassNameFromIndex(int index)
+{
+  if (index < mCTPClasses.size()) {
```

Comment:
this usualy leads to a warning on comparing signed and unsigned, please cast to `index < (int)mCTPClasses.size()`

Reply:
done